lib/create/index.js

```diff
@@ -0,0 +1,73 @@
+const {
+  GitHubUser,
+  Subscription,
+} = require('../models');
+
+const createIssueDialog = require('../messages/create/create-issue-dialog');
+
+async function openDialog(req, res) {
+  const {
+    command,
+    gitHubUser,
+    slackWorkspace,
+  } = res.locals;
+
+  const subscriptions = await Subscription.findAll({
+    where: { channelId: req.body.channel_id, slackWorkspaceId: slackWorkspace.id },
+  });
+  if (subscriptions.length === 0) {
+    return command.respond({ text: 'This channel is not subscribed to any repositories' });
+  }
+
+  let repositories = await Promise.all(subscriptions.map(async (subscription) => {
+    const github = gitHubUser.client;
+    try {
+      const repository = await github.repos.getById({ id: subscription.githubId });
+      return repository.data;
+    } catch (err) {
+      req.log.error({ err, repoId: subscription.githubId }, 'Could not find repository for subscription');
+      if (err.code !== 404) {
+        throw err;
+      }
+    }
+  }));
+  // remove undefined
+  repositories = repositories.filter(repo => repo);
+
+  const { trigger_id } = req.body;
+  await slackWorkspace.client.dialog.open({ dialog: createIssueDialog(repositories), trigger_id });
+
+  return command.respond();
+}
+
+async function dialogSubmit(req, res) {
+  const { slackUser, slackWorkspace } = res.locals;
+
+  const { title, body } = req.body.submission;
+  let { repository } = req.body.submission;
+
+  if (!repository) {
```

Comment:
When will this not be set?

Reply:
When there is just one repo subscribed in the channel, then we default to that one

Reply:
Having said that I wonder if we could have a message menu with just one repository. That might be more consistent both for users and in the implementation 👍